commitments/mode.go

```diff
@@ -31,22 +43,76 @@ func StringToCommitmentMode(s string) (CommitmentMode, error) {
 	}
 }
 
+func StringToEncodingType(s string) (EncodingType, error) {
+	switch s {
+	case "rlp", "RLP", "0":
```

Comment:
maybe use s.toLower() instead?

Reply:
this would also allow Rlp for example

Reply:
04dba8c7e4368cfd65fdfc4b9bb95f480a8ec008